docs/crictl.md

```diff
@@ -0,0 +1,160 @@
+CRICTL Client User Guide
+========================
+This document presumes you already have `containerd` and `cri-containerd`
+installed and running.
+
+This document is for developers who wish to debug/inspect and manage their pods,
+containers, and container images.
+
+Before generating issues against this document, `containerd`, `cri-containerd`,
+or `crictl` please make sure the issue has not already been submitted.
+
+## Install crictl
+If you have not already installed crictl please install the version compatible
+with the `cri-containerd` you are using. The current version is specified
+[here](./hack/version). A helper command has been included to install the
+dependencies at the right version for you:
+```shell
+# make install.deps
```

Comment:
Can we use `console` + `$` instead of `shell` + `#` here?
`#` is treated as a comment in shell.

Reply:
done